setup.py

```diff
@@ -55,7 +55,6 @@
       package_data={'pfnet':['libpfnet*']},
```

Comment:
It could be modified as follows: ```package_data = {'pfnet': ['libpfnet *', '*.dll']}``` to avoid ```ImportError: DLL load failed: The specified module could not be found```  when you use pfnet outside the PFNET.py folder. since pfnet.dll is not copied to the ```Python/Lib/ site-packages/PFNET.egg``` folder.